docs/create-packages/set-package-type.md

```diff
@@ -15,6 +15,8 @@ With NuGet 3.5+, packages can be marked with a specific *package type* to indica
 
 - `DotnetTool` type packages are extensions to the [dotnet CLI](/dotnet/articles/core/tools/index) and are invoked from the command line. Such packages can be installed only in .NET Core projects and have no effect on restore operations. More details about these per-project extensions are available in the  [.NET Core extensibility](/dotnet/articles/core/tools/extensibility#per-project-based-extensibility) documentation.
 
+- `Template` type packages provide [custom templates](/dotnet/core/tools/custom-templates) that can be used to create files or projects like an app, service, tool, or class library.
```

Comment:
Make sure this link works by clicking it on the test deployment -- I've had these relative links act weird before.